wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/wdlom2wom/graph/WorkflowGraphElementToGraphNode.scala

```diff
@@ -46,18 +47,17 @@ object WorkflowGraphElementToGraphNode {
       }
 
     case se: ScatterElement =>
-      val scatterMakerInputs = ScatterNodeMakerInputs(se, a.linkableValues, a.linkablePorts, a.availableTypeAliases, a.workflowName, a.insideAScatter)
+      val scatterMakerInputs = ScatterNodeMakerInputs(se, a.linkableValues, a.linkablePorts, a.availableTypeAliases, a.workflowName, a.insideAScatter, a.tasks)
       ScatterElementToGraphNode.convert(scatterMakerInputs)
 
     case ie: IfElement =>
-      val ifMakerInputs = ConditionalNodeMakerInputs(ie, a.linkableValues, a.linkablePorts, a.availableTypeAliases, a.workflowName, a.insideAScatter)
+      val ifMakerInputs = ConditionalNodeMakerInputs(ie, a.linkableValues, a.linkablePorts, a.availableTypeAliases, a.workflowName, a.insideAScatter, a.tasks)
       IfElementToGraphNode.convert(ifMakerInputs)
+
+    case ce: CallElement =>
+      val callableNodeMakerInputs = CallableNodeMakerInputs(ce, a.tasks, a.linkableValues, a.linkablePorts, a.availableTypeAliases, a.workflowName, a.insideAScatter)
+      CallElementToGraphNode.convert(callableNodeMakerInputs)
   }
 }
 
-final case class GraphNodeMakerInputs(node: WorkflowGraphElement,
-                                      linkableValues: Map[UnlinkedConsumedValueHook, GeneratedValueHandle],
-                                      linkablePorts: Map[String, OutputPort],
-                                      availableTypeAliases: Map[String, WomType],
-                                      workflowName: String,
-                                      insideAScatter: Boolean)
+final case class GraphNodeMakerInputs(node: WorkflowGraphElement, linkableValues: Map[UnlinkedConsumedValueHook, GeneratedValueHandle], linkablePorts: Map[String, OutputPort], availableTypeAliases: Map[String, WomType], workflowName: String, insideAScatter: Boolean, tasks: ErrorOr[Set[CallableTaskDefinition]])
```

Comment:
Personal preference I guess, but when the lists are this long I do tend to prefer splitting them over many lines for readability